spec/active_record/monetizable/shared_contexts.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.shared_context "monetizable product setup" do
```

Comment:
Could we simply copy this product in the two locations it is used? I find that specs are easier to understand when their context is right under your nose, even if it leads to duplication.